components/toast_wrapper/toast_wrapper.jsx

```diff
@@ -90,7 +91,7 @@ class ToastWrapper extends React.PureComponent {
 
         // show unread toast on mount when channel is not at bottom and unread count greater than 0
         if (typeof showUnreadToast === 'undefined' && props.atBottom !== null) {
-            showUnreadToast = unreadCount > 0 && !props.atBottom;
```

Comment:
Is there a reason to remove this check "!props.atBottom" ? I guess it might help resolving the bug?

Reply:
previously it is present as 
```
 if (typeof showUnreadToast === 'undefined' && props.atBottom !== null) {
            showUnreadToast = unreadCount > 0 && !props.atBottom;
        }
````
i have changed to 
```
// show unread toast on mount when channel is not at bottom and unread count greater than 0
        if (typeof showUnreadToast === 'undefined' && props.atBottom !== null) {
            showUnreadToast = unreadCount > 0 && props.initScrollOffsetFromBottom > THRESHOLD_FROM_BOTTOM;
        }

```
which will get the solution for getting the toast from the bottom at 1000px